_episodes/03-types.md

```diff
@@ -47,38 +47,67 @@ end
 ### Types and hierarchy
 
 Here `::Float64` is a type specification, indicating that this variable should
-be a 64-bit floating point number.
+be a 64-bit floating point number, and __`::`__ is an ***operator*** that
+is read "is an instance of."
 If Melissa hadn't specified the type, the variables would have the type `Any`
 by default.
 
-In Julia every type can have only one supertype, so lets check how many types
-are between `Float64` and `Any`
-
-~~~
-julia> supertype(Float64)
-AbstractFloat
-julia> supertype(AbstractFloat)
-Real
-julia> supertype(Real)
-Number
-julia> supertype(Number)
-Any
-~~~
-{: .language-julia}
-
-So we have the relationship `Float64 <: AbstractFloat <: Real <: Number <:
-Any`, where `<:` means "subtype of".
+In Julia every type can have only one supertype, so let's count how many types
+are between `Float64` and `Any`:
+
+1. ~~~
+   supertype(Float64)
+   ~~~
+   {: .language-julia}
+   ~~~
+   AbstractFloat
+   ~~~
+   {: .output}
+2. ~~~
+   supertype(AbstractFloat)
+   ~~~
+   {: .language-julia}
+   ~~~
+   Real
+   ~~~
+   {: .output}
+3. ~~~
+   supertype(Real)
+   ~~~
+   {: .language-julia}
+   ~~~
+   Number
+   ~~~
+   {: .output}
+4. ~~~
+   supertype(Number)
+   ~~~
+   {: .language-julia}
+   ~~~
+   Any
+   ~~~
+   {: .output}
+
+So we have the relationship `Float64 <: AbstractFloat <: Real <: Number <: Any`
+where __`<:`__ is the ***subtype operator***, used here to mean the item
+on the left "is a subtype of" the item on the right.
 
 `Float64` is a _concrete_ type, which means that you can actually create
 objects of this type.
-For example `1.0` is a object of type `Float64`.
-We can check this at the REPL:
+For example `1.0` is an object of type `Float64`.
+We can check this at the REPL using either (or both) the
+`typeof` function or the `isa` operator:
```

Comment:
I looked this up before inserting that language. [Per the docs](https://docs.julialang.org/en/v1/base/base/#Core.isa), `isa(x, type)` is a function that can also be called through the infix operator `x isa type`.

Reply:
I just want to make consistent, what we call an operator, since indeed, [most operators are just functions](https://docs.julialang.org/en/v1/manual/functions/#Operators-Are-Functions).
So either we say here `isa` is a function, that can also be called with infix notation or we say it is an operator, glossing over the fact, that can also use it with prefix notation, since that is not that important to the learner

Reply:
To clarify, I would prefer to label things that take arguments explicitly as functions, and those that are implicit (or infix) as operators. E.g.,
```julia
+(a,b) # function
a + b  # operator
```
An enthusiastic instructor can discuss syntactic sugar. To help, I'm adding a link to the docs for both `isa` and `<:`.

Reply:
5f4f77f